docs/docs/running-on-minikube-without-building.md

```diff
@@ -0,0 +1,294 @@
+# Running Magda on Minikube without building Docker images
+
+These instructions will help you to set up Magda without building
+any of the docker images. These instructions were tested on a Mac
+running MacOS 10.15.7 (Catalina).
+
+These instructions install Magda into its own `magda` namespace.
+
+## Prerequisites
+
+(Installed via homebrew unless otherwise noted)
+
+1. minikube
+2. hyperkit
+3. kubernetes-cli
+4. nvm
+
+Optional, but highly recommended:
+
+-   k9s
+
+## Preparation
+
+### Starting Kubernetes
+
+```bash
+minikube config set driver hyperkit
+
+# Currently supported by Magda
+minikube config set kubernetes-version v1.16.5
+minikube config set memory 16384
+minikube config set cpus 4
+minikube config set dashboard true
+minikube config set disk-size 60G
+minikube start
+```
+
+### Starting Magda with the [Building-and-Running](https://magda.io/docs/building-and-running) instructions \*\*
+
+\*\* Minus the building part.
+
+Install the prerequisites on the Building-and-Running page.
+Then:
+
+```bash
+helm repo add magda-io https://charts.magda.io
+helm repo add twuni https://helm.twun.io
+# Get update from repos
+helm repo update
+```
+
+Those instructions are missing these steps (but I guess they're
+implied):
+
+```bash
+git clone https://github.com/magda-io/magda.git
+cd magda
+```
+
+Then, following the instructions:
+
+```bash
+helm install docker-registry -f deploy/helm/docker-registry.yml twuni/docker-registry
+helm repo add mittwald https://helm.mittwald.de
+helm repo update
+kubectl create namespace kubernetes-replicator
+helm upgrade --namespace kubernetes-replicator --install kubernetes-replicator mittwald/kubernetes-replicator
+```
+
+Edit the file at `deploy/helm/minikube-dev.yaml` to comment out these lines:
+
+```yaml
+# image:
+#   repository: "localhost:5000/data61"
+#   tag: "latest"
+#   pullPolicy: Always
+```
+
+Then create secrets (and, for output see the bottom of this file) :
+
+```bash
+nvm use lts/erbium
```

Comment:
Probably `12` is more clear than the code name `erbium`.
And set the `default` to 12
```suggestion
nvm use 12
nvm alias default 12
```